packages/modal/README.md

```diff
@@ -0,0 +1,24 @@
+
+# pcln-slider
+
+React modal component
+
+```sh
+npm i pcln-modal
+```
+
+```jsx
+import Modal from 'pcln-modal'
+```
+
+```jsx
+<Modal
+  isOpen={this.state.isOpen}
+  onClose={handleClose}
+  bg="white"
+  title="Modal title"
```

Comment:
This `title` prop looks like it should be `props.children` – might be worthwhile to expand on the README a bit more to lock down the API before jumping into the implementation

Reply:
Yes, probably finding a chance with Wes and you to figure out what level of flexibility we want to give people and then knock down the API's